### PR TITLE
[PartEditor] Fill Operators listbox

### PR DESCRIPTION
**Pull request by seanshpark**
This will fill Operators listbox from operator names with
calling circle-operator tool.

ONE-vscode-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #641, from draft #647

listbox will be filled something like this;
image

File: src/PartEditor/PartEditor.ts [hunk 1]
```diff
@@ -123,6 +130,58 @@ export class PartEditorProvider implements vscode.CustomTextEditorProvider {
     }
   }
 
+  private handleRequestOpNames() {
+    const K_DATA: string = 'data';
+    const K_EXIT: string = 'exit';
+    const K_ERROR: string = 'error';
+    // TODO integrate with Toolchain
```
Comment (jyoungyun) [inline src/PartEditor/PartEditor.ts:137]: 👀

File: src/PartEditor/PartEditor.ts [hunk 2]
```diff
@@ -123,6 +130,58 @@ export class PartEditorProvider implements vscode.CustomTextEditorProvider {
...
+          resolve(result);
+        } else {
+          let msg = 'Failed to load model: ' + this._modelFileName;
+          Balloon.error(msg);
+          reject(msg);
+        }
+      });
+
+      cmd.on(K_ERROR, (err) => {
+        Balloon.error('Failed to run circle-operator: ' + this._modelFileName);
```
Comment (jyoungyun) [inline src/PartEditor/PartEditor.ts:170]: (Q) Is it okay not to call `reject()` here?

Comment (seanshpark) [reply src/PartEditor/PartEditor.ts:170]: Thanks for this, I'll check.

Comment (seanshpark) [reply src/PartEditor/PartEditor.ts]: added reject()

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**